buildkite/build_project.sh

```diff
@@ -206,10 +210,16 @@ case "$REPO_FULL_NAME" in
         ;;
 
     ldc-developers/ldc)
-        git submodule update --init
+        cd runtime && git clone --depth 5 https://github.com/ldc-developers/druntime
+        git clone --depth 5 https://github.com/ldc-developers/phobos
+        cd .. && git submodule update
```

Comment:
Alright, this should address your concerns.

Reply:
Unrelated CI failure, LDC builds fine.

Reply:
The failure was due to regression that managed to sneak into Phobos - see e.g. https://github.com/dlang/ci/pull/286

I restarted the build.